static/app/components/onboardingWizard/progressHeader.tsx

```diff
@@ -29,10 +39,8 @@ function ProgressHeader({allTasks, completedTasks}: Props) {
           color: ${theme.textColor};
         `}
       />
-      <HeaderTitle>{t('Quick Start')}</HeaderTitle>
-      <Description>
-        {t('Walk through this guide to get the most out of Sentry right away.')}
-      </Description>
+      <HeaderTitle>{tct('[title]', {title})}</HeaderTitle>
```

Comment:
@roggenkemper ok I see you have it here. This doesn't work because our transaction tool is looking for literal strings in a `t` or `tct`. So you need to do it like:
```
title = t('Quick Start');
```

Reply:
Yeah this would present the translators with the string `[title]` to translate, which is nothing to translate hah